src/main/c/Makefile

```diff
@@ -24,6 +24,7 @@ libbwa.$(LIB_EXT): $(JNI_BASE_NAME).o jnibwa.o bwa/libbwa.a
 
 bwa:
 	git clone https://github.com/lh3/bwa && cd bwa && git checkout $(BWA_MEM_COMMIT) && echo '#define BWA_COMMIT "'$(BWA_MEM_COMMIT)'"' > bwa_commit.h
+	sed -i .bak 's/\(LOBJS=.*\)/\1 $$(AOBJS)/g' bwa/Makefile
```

Comment:
Also I have spotted what looks like a memory leak in ```..._BwaMemIndex_createIndexImageFile``` we do not ```free``` the result of the ```strdup``` calls. Moreover there is no need to call strdup as long as one release the first copies  with ```ReleaseStringUTFChars``` after the work is done.